src/index.ts

```diff
@@ -317,6 +317,19 @@ export default class CodeTool implements BlockTool {
         case 'Tab':
           this.tabHandler(event);
           break;
+        case 'Backspace':
+          if (textarea.value.length > 0) {
+            event.stopPropagation();
+          } else {
```

Comment:
please, explain this case

Reply:
## `event.stopPropagation()`
This keeps `backspace` and `cursor` keys from triggering other key event listeners in the application or editor that might interfere with navigation within the textarea.

Reply:
I read somewhere that in Firefox `backspace` could navigate back to previous page and it was removed in the recent versions (3 years ago 😬I guess). I will remove that as it doesn't has any impact.